include/benchmark/benchmark.h

```diff
@@ -429,6 +429,46 @@ struct Statistics {
       : name_(name), compute_(compute) {}
 };
 
+// Converts whatever the stored manual time is to
+// seconds, which is used to estimate iterations
+// and similar
+typedef double(BenchmarkTimeCostFunc)(double);
+
+struct BenchmarkTime {
```

Comment:
it has state, a private section, a user-defined constructor: it's a `class`, not a `struct`.